bundles/core-combat/player-events.js

```diff
@@ -74,8 +74,8 @@ module.exports = (srcPath) => {
             const damage = Math.min(playerDamage, target.getAttribute('health'));
 
             target.emit('hit', this, damage);
-            const startingHealth = target.getRawAttribute('health');
-            target.setAttribute('health', startingHealth - damage);
+            const startingHealth = target.getAttribute('health');
```

Comment:
can probably delete the `startingHealth` variables since they're not used anymore